frontend/app/components/mapbox/current-mouse-position.js

```diff
@@ -1,16 +1,38 @@
 import Component from '@ember/component';
-import { action } from '@ember-decorators/object';
+import { action, computed } from '@ember-decorators/object';
 
 export default class MapboxCurrentMousePositionComponent extends Component {
   currentMapMouseEvent = null;
 
   @action
   handleMouseMove(e) {
     this.set('currentMapMouseEvent', e);
+    this._passPositionToParent();
   }
 
   @action
   handleMouseLeave() {
     this.set('currentMapMouseEvent', null);
+    this._passPositionToParent();
   }
+
+
+  // for yielding mousePosition in template
+  @computed('currentMapMouseEvent')
+  get mousePosition(){
+    return this._pointOfMouseEvent(this.currentMapMouseEvent);
+  }
+
+  _passPositionToParent(){
+    if(this.handleMousePosition){
+      let mousePoint = this._pointOfMouseEvent(this.currentMapMouseEvent);
+      this.handleMousePosition(mousePoint);
```

Comment:
so `handleMousePosition` is an argument ? let's make sure it has a default value and comment explaining it is an event hook passed down from the parent component?

Reply:
No I really like the nit, I'm going to amend and push again with the comments.